### PR TITLE
Update osu!tourney page with latest bancho changes.

### PR DESCRIPTION
**Pull request by smoogipooo**
More changes will come after Bancho is re-updated.

### CODE REVIEW

File: wiki/osu!tourney/en.md [hunk 1]
```diff
@@ -207,8 +206,9 @@ Sending `!mp help` to BanchoBot will reveal the above commands.
 
 The following is an example of command usage:
 
-- `!mp move Zallius 1` - Move Zallius into slot 1 of the room.
-- `!mp move Loctav 4` - Move Loctav into slot 4 of the room.
+- `!mp invite Zallius` - Invite Zallius to the room.
```
Comment (peppy) [inline wiki/osu!tourney/en.md]: why is this here twice?

Comment (smoogipooo) [inline wiki/osu!tourney/en.md]: Fixed (although this comment wasn't marked as resolved).
